docs/zk2kraft.html

```diff
@@ -155,14 +155,15 @@ <h5 class="anchor-heading">Removal metrics</h5>
         </li>
         <li>
             <p>
-                Remove the metrics which is monitoring the latency in milliseconds for ZooKeeper requests from broker.
+                Remove the metrics which are only used in ZooKeeper mode.
             </p>
             <ul>
                 <li><code>kafka.server:type=ZooKeeperClientMetrics,name=ZooKeeperRequestLatencyMs</code></li>
+                <li><code>kafka.controller:type=ControllerChannelManager</code></li>
```

Comment:
Should we list complete metrics attributes? It would be useful to search specific metrics. WDYT?

Reply:
#18658 add complete metrics, so I think PR should follow that pattern too.

Reply:
I will follow it. Thanks.

Reply:
Updated it. Thanks.

Reply:
Should we merge all metrics item into single one?

Reply:
there are three listed items for now